test/helpers/clone.test.js

```diff
@@ -148,9 +148,11 @@ describe('clone', () => {
 
     describe('constructor is RegExp', () => {
       it('return new equal date ', () => {
-        const base = new RegExp(/A-Z.*/);
+        const base = new RegExp(/A-Z.*/g);
+        base.lastIndex = 2;
         const cloned = clone(base);
         assert.deepStrictEqual(base, cloned);
```

Comment:
Would be great if we could assert that the flags get copied over.
Not sure if `deepStrictEqual` checks for that.

Reply:
I tested it. It checks for the flags. But it does not check for lastIndex. Thats why I added the separate lastIndex assertion.